MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -13,6 +13,8 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		didSet { tableView.reloadData() }
 	}
 
+	@IBOutlet var errorView: ErrorView!
```

Comment:
This setter can be private to prevent clients from changing the instance:

```suggestion
	@IBOutlet private(set) var errorView: ErrorView!
```

Reply:
it is implemented